apps/angular/projection/src/app/ui/card/card.component.html

```diff
@@ -1,27 +1,17 @@
 <div
-  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3"
-  [class]="customClass">
-  <img
-    *ngIf="type === CardType.TEACHER"
-    src="assets/img/teacher.png"
-    width="200px" />
-  <img
-    *ngIf="type === CardType.STUDENT"
-    src="assets/img/student.webp"
-    width="200px" />
+  class="border-2 border-black rounded-md p-4 w-fit flex flex-col gap-3 bg-inherit">
```

Comment:
Is bg-inherit part of tailwind?

Reply:
yep, its just applying `background-color: inherit`
https://tailwindcss.com/docs/background-color#:~:text=Preview-,bg%2Dinherit,-background%2Dcolor%3A